### DISCUSSION TITLE
Road to 1.0.0

### DISCUSSION THREAD

**Discussion by cdmoro** [cdmoro/bootstrap-vue-3#158] [General]
Hello there, I consider the project is mature enough to think in the next step: a production-ready version. I want to discuss with you (contributors and users) what it takes to achieve this milestone.

In a general view, we should take care of these three subjects:
- Documentation
- Testing
- Components

Let me know what you think :blush:!

**Comment by itf2012** [comment]
Hi @cdmoro,

I got on this site to try to find out if there were any timelines mentioned for a production-ready version of Bootstrap Vue 3 and found this conversation. Am I reading this correct that you think Bootstrap Vue 3 is at, or very close to, releasing a production-ready version?

If that is not the case, is there anything anywhere one can look to see a general timeline as to the progress and projected releases? We are currently starting a brand new project and trying to determine whether we can wait for a production ready version of Bootstrap Vue 3 or if we need to use something else.

Thanks,
Ryan G.

**Comment by jclijmans** [comment]
For me the most valuable component of the bootstrap-vue is the BTable component with sorting, paging.
At the same time, I realize that this is a very complex component to write.

I think we should give priority on implementing this.

**Comment by cdmoro** [comment]
You are right @jclijmans, BTable is THE component. I think we could make a pool or something in order to know what are the minimum props we need to have a first version of the table. In the next releases we can add more and more props.

**Comment by xvaara** [comment]
Is the goal to be compatible with BootstrapVue or to "make it better"?

**Comment by jclijmans** [comment]
In my opinion we should start by focusing on compatibility. That's already a huge task.
After that we can implement new stuff.

During development, we should of course use the latest vue3 with typescript and preferable the composition api.

**Reply by cdmoro** [reply]
I'm totally agree with that
